broadinstitute/consent
# [DUOS-1631][risk=no] Send Submitted For Review Email For DAR Collections

## Addresses
https://broadworkbench.atlassian.net/browse/DUOS-1631

## Changes
Updated the logic around sending emails for every DAR in a collection. Instead, we send a single message about the collection to each user in the DAC. 

## Testing
* Set up some number of test users in a DAC
* Assign some datasets to that DAC, or use a DAC that already has datasets
* Set your local mail configuration in consent to true: `activateEmailNotifications: true`
* As a researcher user, create a DAR against those datasets. **Important** that there be more than one so there will be multiple DARs in the collection
* As an admin or chair, visit the new chair console page and create an election for the submitted collection. **Important** that you visit the new console page - the old one only creates one election per DAR.
* Check that your DAC user received exactly 1 email for the submitted collection. Previously, you would have received N emails, one per DAR in the collection.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment
